pkg/ddl/job_table.go

```diff
@@ -404,7 +404,16 @@ func (d *ddl) delivery2LocalWorker(pool *workerPool, task *limitJobTask) {
 			metrics.DDLRunningJobCount.WithLabelValues(pool.tp().String()).Dec()
 		}()
 
-		err := wk.HandleLocalDDLJob(d.ddlCtx, job)
+		maxRetryTime := 10
+		var err error
+		for i := 0; i < maxRetryTime; i++ {
+			err = wk.HandleLocalDDLJob(d.ddlCtx, job)
+			if err == nil || !isRetryableError(err) {
```

Comment:
I remember that if the job itself is not finished (`job.IsFinished` is false), it will retry itself. Why is there a special emphasis here?

Reply:
for fast create table, the job does not exist in the ddl job table, so the original retry mechanism (taken out from the ddl job table and re-executed) will not take effect.

Reply:
Got it, could we add a comment here?